po/de.po

```diff
@@ -1048,8 +1053,8 @@ msgid ""
 "Details for machine {{ machineid }} in Machine Resolver\n"
 "        {{ machineresolver }}"
 msgstr ""
-"Details zur Maschine {{ machineid }} im Maschinen-Resolver "
-"{{ machineresolver }}"
+"Details zur Maschine {{ machineid }} im Maschinen-Resolver {{ "
+"machineresolver }}"
```

Comment:
This looks like an evil linebreak in the template. I am not sure, if this will work out!